packages/react-server/core/__tests__/integration/title/AsyncServerTimeoutTitlePage.js

```diff
@@ -1,10 +1,10 @@
-var Q = require("q");
+var Promise = require("bluebird");
 
 class AsyncServerTimeoutTitlePage {
 	getTitle() {
 		// this is a title that should NOT display on the server, as it's more than the server timeout.
-		return Q("An asynchonous timeout title").delay(500);
+		return Promise.resolve("An asynchonous timeout title").delay(500);
```

Comment:
As I understand it, we require our clients to use the same promise library that we use internally (which makes sense), which means that they need to take their own dependency on bluebird.  A week ago, I'd have said that's not a problem, since we'll always be on the same promise library, yet here we are.  Would it make sense to export an austensibly custom promise implementation that is just bluebird stripped down to the parts of the api that we actually use and then guarantee that if you use that implementation (as stored in the monorepo), that you'll always be able to upgrade cleanly between react-server versions (even if we change promise implementations under the covers)?  Or to use a peerDependency to communicate that clients need to use bluebird (and which versions we support)?